src/components/Layout/Page.tsx

```diff
@@ -19,7 +21,10 @@ function TopOfTopSection(props: { background?: React.CSSProperties["background"]
     )
   } else if (platform === "ios") {
     // Add some additional v-space for the iPhone X notch
-    return <div className="iphone-notch-top-spacing" />
+    return isSmallScreen ? <div className="iphone-notch-top-spacing" /> : <></>
```

Comment:
Shouldn't it show the top padding on larger iPad screens as well?

Reply:
No, the `isSmallScreen` is necessary because for example dialogs like the TransactionReview are not shown in fullScreen mode on larger screens so that without this check the 'non-fullscreen' dialogs would have too much padding on top.